stages/test/test_insights_client_conf.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python3
+
+STAGE_NAME = "org.osbuild.insights-client.config"
+
+CONF_FILE = """[insights-client]
+# Example options in this file are the defaults
+
+# Change log level, valid options DEBUG, INFO, WARNING, ERROR, CRITICAL. Default DEBUG
+#loglevel=DEBUG
+
+# Attempt to auto configure with Satellite server
+#auto_config=True
+
+# URL for your proxy.  Example: http://user:pass@192.168.100.50:8080
+#proxy=
+"""
```

Comment:
Let's make the conf file in this test a bit more complex. I suggest adding both a commented-out and non-commented out thing with the same name and to add a value that is supposed to be changed.

For example:
```
#proxy=
proxy=https://localhost/
```

To see if it correctly replaces this.